eval/mt_bench/components.py

```diff
@@ -188,14 +188,16 @@ def shutdown_vllm(process: subprocess.Popen, timeout: int = 20):
         scores[model_path] = overall_score
 
     with open(output_path, "w", encoding="utf-8") as f:
-        json.dump(all_mt_bench_data, f, indent=4)
+        for data in all_mt_bench_data:
+            json.dump(data, f)
+            f.write("\n")
 
     outputs = NamedTuple("outputs", best_model=str, best_score=float)
     best_model = max(scores, key=scores.get)
     best_score = scores[best_model]
-    if best_score_file:
-        with open(best_score_file, "w", encoding="utf-8") as f:
-            json.dump({"best_model": best_model, "best_score": best_score}, f, indent=4)
+
+    with open(best_score_file, "w", encoding="utf-8") as f:
+        json.dump({"best_model": best_model, "best_score": best_score}, f, indent=4)
```

Comment:
If we are creating a new file, would it be worth uploading it as an artifact as well? (like we do with the other file via `pvc_to_mt_bench_op` task)

Reply:
We could. Seems like its somewhat duplicative information though. Might make sense to just insert this information into the top of the `output/mt_bench_data.json` instead and keep it all in a single file. WDYT?

Reply:
I went ahead and consolidated all mt_bench info into `mt_bench_data.json` so no longer any need to mange this extra file.